cmd/vclusterctl/cmd/create.go

```diff
@@ -94,14 +96,15 @@ vcluster create test --namespace test
 
 // Run executes the functionality
 func (cmd *CreateCmd) Run(ctx context.Context, args []string) error {
-	manager, err := platform.GetManager(cmd.Manager)
-	if err != nil {
-		return err
-	}
+	cfg := cmd.GlobalFlags.LoadedConfig(cmd.log)
 
+	// check if there is a platform client or we skip the info message
+	_, err := platform.NewClientFromPath(ctx, cmd.Config)
```

Comment:
Had to move this outside of PintManagerInfo, otherwise it would cause an import cycle.

Reply:
Thats okay, we can also move the function into the platform package to avoid that